### PR TITLE
Refactor cmd tool to support mutil project

### PR DESCRIPTION
**Pull request by JiepengTan**

### CODE REVIEW

File: cmd/gox/go.mod
Comment (nighca) [inline cmd/gox/go.mod:1]: Is there any concern to rename the cmd tool from "spx" to "gox"? "gox" does not seem related to spx, and [gogen](https://github.com/goplus/gogen/commit/7f2b2dac739de284737f83c2ee03de165baad62b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1) used to be named with "gox" 🤣

Comment (JiepengTan) [reply cmd/gox/go.mod:1]: it doesn't matter , this tool would be install as spx 
```
appname="spx"
# install cmd
go build -o $appname
mv $appname $(go env GOPATH)/bin/

# install igox
spx setupweb
```

Comment (nighca) [reply cmd/gox/go.mod:1]: OK

**Review by nighca — APPROVED** [review]
